samples/switchless/switchless.edl

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+enclave {
+    trusted {
+        public void enclave_helloworld();
+    };
```

Comment:
Does it make sense to also show transition_using_threads support on the ecall?

Reply:
Switchless ecall hasn't been implemented yet. We will update this sample once it's added.